client/blocks/inline-help/test/admin-setions.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { filterListBySearchTerm } from 'blocks/inline-help/admin-sections';
+
+describe( 'filterListBySearchTerm()', () => {
+	const mockCollection = [
+		{
+			title: 'The best section',
+			description: 'Better than that other section.',
+			link: `/best/section/eva`,
+			synonyms: [ 'yolo', 'gud' ],
+			icon: 'beta',
+		},
+		{
+			title: 'Quite a good page',
+			description: 'A rather agreeable page if you ask me.',
+			link: `/acceptable/page`,
+			synonyms: [ 'hi', 'bye' ],
+			icon: 'gamma',
+		},
+	];
+
+	test( 'should ignore non-word characters and return `[]`', () => {
```

Comment:
Personally I still think that code should not be included in a test description. If I'm a non-technical person I should be able to read the test case and fully understand it.

Many examples of this throughout the tests 🤷